packages/plugin-assets-retry/src/runtime/initialChunkRetry.ts

```diff
@@ -246,7 +246,12 @@ function retry(config: RuntimeRetryOptions, e: Event) {
   // if the initial request is "/static/js/async/src_Hello_tsx.js?q=1", retry url would be "/static/js/async/src_Hello_tsx.js?q=1&retry=1"
   const originalQuery =
     target.dataset.rsbuildOriginalQuery ?? getQueryFromUrl(url);
-  function getUrlRetryQuery(existRetryTimes: number): string {
+
+  // this function is same with async chunk retry
+  function getUrlRetryQuery(
+    existRetryTimes: number,
+    originalQuery: string,
```

Comment:
Is this `originalQuery` param the same as `const originalQuery` in L247?

Reply:
yes, these two functions are exactly the same now

Reply:
But the `originalQuery` param seems to be unnecessary in this file